.github/workflows/unitest.yml

```diff
@@ -0,0 +1,21 @@
+name: Unit test
```

Comment:
it doesn't specifically seem like unit tests. we are running all test files in this workflow, let's rename it as just Test or specify unit tests with its folder or however we store it

Reply:
It runs all .spec files in the src folder, for e2e test I believe there is different folder, having that it runs unit test? Or Am I missing something?

Reply:
Please Feel free to reopen the comment if I missed something, 
Thank you

Reply:
Got it, thanks for review, updated